doc/source/cluster/running-applications/job-submission/quickstart.rst

```diff
@@ -111,12 +111,19 @@ Make sure to specify the path to the working directory in the ``--working-dir``
     # Job 'raysubmit_inB2ViQuE29aZRJ5' succeeded
     # ------------------------------------------
 
-This command will run the script on the Ray Cluster and wait until the job has finished. Note that it also streams the stdout of the job back to the client (``hello world`` in this case). Ray will also make the contents of the directory passed as `--working-dir` available to the Ray job by downloading the directory to all nodes in your cluster.
+This command will run the entrypoint script on the Ray Cluster's head node and wait until the job has finished. Note that it also streams the stdout of the job back to the client (``hello world`` in this case). Ray will also make the contents of the directory passed as `--working-dir` available to the Ray job by downloading the directory to all nodes in your cluster.
 
 .. note::
 
     The double dash (`--`) separates the arguments for the entrypoint command (e.g. `python script.py --arg1=val1`) from the arguments to `ray job submit`.
 
+.. note::
+
+    By default the entrypoint script is run on the head node. To override this, specify any of the arguments 
```

Comment:
"entrypoint script is run on the head node" => Do you mean the driver process would be running on the head node by default?

Reply:
Yes, we say `entrypoint script` here to convey that it is running whatever the user specifies as `entrypoint`.  Typically this is a script that starts a Ray driver process (`ray.init()`), but it could also be any command at all, like `echo hello && pip install something`. It technically doesn't have to involveRay

Reply:
Thanks!